includes/subscriptions.php

```diff
@@ -238,24 +238,24 @@ function send_notifications( $post_id ) {
 		return;
 	}
 
-	if ( ! wp_doing_cron() ) { // @codingStandardsIgnoreLine `wp_doing_cron(..)` is a WP function
+	if ( ! wp_doing_cron() ) { //phpcs:ignore
+		if ( ! current_user_can( 'edit_post', $post_id ) ) {
```

Comment:
Subscriptions are run in the cron context, will this user check work in cron? I suspect not.

Reply:
Not sure what do you mean, there is no 'current user' in cron, so we need the `current_user_can(..)` check when 'is not doing cron'.